launcher.ini

```diff
@@ -1,5 +1,5 @@
 [launcher]
-#interactive = true
+interactive = true
```

Comment:
I'm not sure I want to make everyone that uses the launcher default to be interactive. Can we just copy over a launcher-docker.ini into place and I'll add a launcher-docker.ini to the repo?